app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,24 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var MAILCHECK_ON = 'true';
+      var abChoose;
+      var abData = {
+        isMetricsEnabled: !! this.metrics._able,
+        uuid: this.metrics._uuid
+      };
+
+      var mailcheckParamChoice = Url.searchParam('mailcheck', this.window.location.search);
+      // the window parameter will override any ab testing features
```

Comment:
I think force flags should be sent to Able. It might be nice to have a convention for doing it automatically so we don't need to think about them in the experiment code, but in the meantime a force flag should force eligible true and the choice to its value.